examples/dreambooth/train_dreambooth_lora_sdxl.py

```diff
@@ -408,6 +408,12 @@ def parse_args(input_args=None):
         default=4,
         help=("The dimension of the LoRA update matrices."),
     )
+    parser.add_argument(
+        "--scale_scheduler",
+        action="store_true",
+        default=False,
+        help="Rescale Scheduler based on Zero SNR paper during training and sampling",
```

Comment:
Let's also add the link for reference here.

Reply:
i would prefer the argument be `--rescale_betas_zero_snr`